deploy/setup-project.sh

```diff
@@ -3,17 +3,40 @@
 set -o nounset
 set -o errexit
 
+readonly PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
+readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
+
+BIND_ROLES="roles/compute.storageAdmin roles/iam.serviceAccountUser projects/${PROJECT}/roles/gcp_compute_persistent_disk_csi_driver_custom_role"
 IAM_NAME="${GCEPD_SA_NAME}@${PROJECT}.iam.gserviceaccount.com"
 
-# Cleanup old Service Account and Key
+# Create or Update Custom Role
+if gcloud iam roles describe gcp_compute_persistent_disk_csi_driver_custom_role --project "${PROJECT}";
+then
+  yes | gcloud iam roles update gcp_compute_persistent_disk_csi_driver_custom_role \
```

Comment:
does the -q option not work for suppressing interactive mode?

Reply:
brainfarted